docs/cloud/datastores.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar_position: 1
+---
+
+# Data Stores
+
+A Dragonfly Cloud data store represents a Redis Protocol (RESP) endpoint.
+
+To create a data store, on the *Data Stores* tab, click [+Data Store](https://dragonflydb.cloud/datastores/new)
+
+The minimum configuration consists of *name*, *cloud provider*, *cloud region* and *plan*.
+
+Note that the *cloud provider* can not be modified once the data store is created.
+
+The *plan* specifies the provisioned memory and memory to cpu ratio of the data store.
+
+You can modify the data store *plan* later with zero down time to scale it up or down.
+
+By default the data store will be configured with a *public endpoint*, *TLS* and an auto generated *passkey*, meaning you can securely connect to it from anywhere over the public internet.  
+To create a data store with a *private endpoint*, see [Security](#security), [Networks](./networks.md) and [Peering Connections](./connections.md).
+
+By default the data store will consist of a single Dragonfly server, to create a highly available data store see [High Availability](#high-availability).
+
+Once the data store is created, clicking the data store row will open a drawer with the data store configuration, including the auto generated passkey and a redis compatible *connection string*. 
+
+Once the data store *Status* becomes *Active* you can try access it with e.g. `redis-cli -u <connection string> PING`
+
+To update the data store configuration click the pencil edit button in the top right of the drawer.  
+Dragonfly cloud performs data store updates with zero downtime.    
+
+## Security 
+Dragonfly Cloud supports public and private endpoints
+
+### Public Endpoint 
+With a public endpoint you can connect to the data store from anywhere over the internet. 
+To protect your data store from unauthorized access public endpoints are configured with *passkey* and *TLS* enabled.   
+*TLS* has some performance impact so can be disabled but it is highly recommended to leave it enabled.  
+*Passkey* is mandatory and can not be disabled for public endpoints 
+
+### Private Endpoint 
+*Private endpoint* provides better security, performance and lower latency as the data transports to and from the data stores over private networks and not via the public internet.  
+It also reduces data transfer costs applied by the cloud provider.
```

Comment:
It also reduces data transfer costs applied by the cloud provider.
-> Using a private endpoint also reduces data transfer costs applied by the cloud provider.